### ISSUE TITLE
[PROPOSAL] Resizing windows shortcuts optimization

### ISSUE THREAD

**Issue by dodalovicgran** [pop-os/cosmic-epoch#1401]
We have `SUPER+R` and `SUPER+SHIFT+R`. 
We could equally so have just `SUPER+R` and then just use left and right arrows or  `h` and `l` keys.

**Comment by git-f0x** [comment]
How would you choose the window edge to be resized, since all 4 arrows are needed to be able to choose the edge, especially when tiling?

**Comment by dodalovicgran** [comment]
You're right, this is needed when you have many windows tiled. Although, when you have for instance 2 of them, it's unnecessarily complex.

**Comment by mmstick** [comment]
A model mode was initially discussed, but we decided to use the current approach in the end.
